src/documents/views.py

```diff
@@ -444,6 +445,24 @@ def get_metadata(self, file, mime_type):
             logger.warning(f"No parser for {mime_type}")
             return []
 
+    def get_pages_count(self, file, mime_type):
+        if not os.path.isfile(file):
+            return None
+
+        parser_class = get_parser_class_for_mime_type(mime_type)
+        if parser_class:
+            parser = parser_class(progress_callback=None, logging_group=None)
+
+            try:
+                return parser.get_pages_count(file)
+            except Exception:  # pragma: no cover
+                logger.exception(f"Issue getting pages count for {file}")
```

Comment:
```suggestion
                logger.exception(f"Error getting pages count for {file}")
```

Nit, obviously :)

And perhaps including the actual error